content/posts/numpy/numpy2/index.md

```diff
@@ -68,26 +68,38 @@ the [NumPy 2 release notes](https://numpy.org/devdocs/release/2.0.0-notes.html).
 
 A major release comes with changes that users may have to adapt to, but we
 worked hard to strike a balance between improvements and ensuring that the
-transition to NumPy 2 is as seamless as possible. We wrote a comprehensive
+transition to NumPy 2.0 is as seamless as possible. We wrote a comprehensive
 [migration guide](https://numpy.org/devdocs/numpy_2_0_migration_guide.html),
 and a [ruff plugin](https://numpy.org/devdocs/numpy_2_0_migration_guide.html#ruff-plugin)
-helps to automatically update Python code so it will work with both NumPy 1 and
-NumPy 2.
+helps to automatically update Python code so it will work with both NumPy 1.x and
+NumPy 2.x.
+
+We knew when we started the development for 2.0 that rolling out a NumPy 2.0
+will be (temporarily) disruptive, because of the backwards-incompatible ABI change
+in particular which prevents previously-released wheels of other packages
```

Comment:
I am fairly sure that the ABI change will end up being the most disruptive, by some distance - it already has been over the past months. For a few projects it'll be NEP 50 (Numba, CuPy), for a few it'll be a C API change, for end users with pure Python code it's both adapting to some Python-level changes as well as to ABI breakage (since they're going to have to manually do quite a bit of dependency management).

But no point arguing about it - if you don't like it, let's change it. I'll push a tweak in a minute.